ethcore/db/src/db.rs

```diff
@@ -36,6 +36,7 @@ pub const COL_EXTRA: u32 = 3;
 /// Column for Traces
 pub const COL_TRACE: u32 = 4;
 /// Column for the empty accounts bloom filter.
```

Comment:
the docs here say for "the empty accounts", but is it actually true?

Reply:
no it's not correct, especially not given the meaning of "empty" in ethereum. Fixing.